wdtk-client/src/test/java/org/wikidata/wdtk/client/ClientTest.java

```diff
@@ -83,6 +83,23 @@ public void testQuietLoggingConfig() throws ParseException, IOException {
 	}
 
 	@Test
+	public void testJsonOutput() {
+		DumpProcessingAction action = new JsonSerializationAction();
+		action.open();
+		action.close();
+		assertEquals(action.getReport(),
+				"Finished serialization of 0 EntityDocuments in file {PROJECT}-{DATE}.json");
+	}
+
+	@Test
+	public void testRdfOutput() {
+		DumpProcessingAction action = new RdfSerializationAction();
+		action.open();
+		action.close();
+		assertEquals(action.getReport(),
+				"Finished serialization of 24 RDF triples in file null");
```

Comment:
This is not a robust test. The number 24 will change as soon as we change the vocabulary.